src/jobservice/config/config.go

```
@@ -120,11 +121,11 @@ func LocalRegURL() (string, error) {
 
 // LogDir returns the absolute path to which the log file will be written
 func LogDir() (string, error) {
-	cfg, err := mg.Get()
-	if err != nil {
-		return "", err
+	dir := os.Getenv("LOG_DIR")
+	if len(dir) == 0 {
+		dir = defaultLogDir
 	}
-	return cfg[comcfg.JobLogDir].(string), nil
+	return dir, nil
```

does it need to return the second err? seems it's always nil.